roles/libvirt_manager/tasks/create_vms.yml

```diff
@@ -51,6 +50,14 @@
         owner: "{{ ansible_user_id }}"
         state: file
 
+    - name: Create cloud-init ISO
+      when:
+        - vm_data.disk_file_name != 'blank'
```

Comment:
we also probably don't want to create the ISO for VM that:
- aren't set to start
- aren't set to be managed

and, therefore, we don't want to attach that ISO with the same conditions. Maybe you can move this task in a common block with the one attaching the ISO, so that you have one `when` block?

Reply:
makes sense, let me try.